src/lib/cozy-client/CozyClient.js

```diff
@@ -61,9 +80,9 @@ export default class CozyClient {
     return this.facade.getAdapter(doctype)
   }
 
-  async fetchCollection(name, doctype, options = {}, skip = 0) {
+  async fetchDocuments(queryName, doctype, options = {}, skip = 0) {
```

Comment:
I think you didn't rename all the previous calls? For exemple there's one in the `SuggestionProvider` that hasn't been changed I think.

Reply:
Good catch, thanks!